### PR TITLE
Update README

### PR DESCRIPTION
**Pull request by mzfr**
Added new features to the README. Also updated `usage` section.

Is there anything else we should add to the README ? Any add-on rules or something else

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -3,35 +3,79 @@
...
 ```
 
-## How to use
+#### For Developement
 
-1. Open Terminal in add-on repository directory
-2. Execute `kodi-addon-checker`
+* Clone the repository
+```
+git clone https://github.com/mzfr/addon-check/
```
Comment (MartijnKaijser) [inline README.md]: should be our repo location

**Comment by MartijnKaijser** [comment]
addon rules should be linked to the wiki page although they are team kodi specific 
https://kodi.wiki/view/Add-on_rules

File: README.md (hunk 1 above)
Comment (mzfr) [reply README.md]: Fixed

File: README.md [hunk 2]
```diff
@@ -3,35 +3,79 @@
 
 # Kodi Addon checker
 
-This tool checks the Kodi repo for best practices and creates problem and warning reports
+This tool checks the Kodi repo for best practices and creates problem and warning reports.
```
Comment (enen92) [inline README.md]: the tool checks not only some best practices but mainly our Addon-rules/submission guidelines (https://kodi.wiki/view/Add-on_rules). The way it is written also implies you are creating 2 different reports (one for problem and another for warning). Use maybe: produces a report containing found problems and warnings.

File: README.md [hunk 3]
```diff
@@ -3,35 +3,81 @@
 
 # Kodi Addon checker
 
-This tool checks the Kodi repo for best practices and creates problem and warning reports
+This tool checks the Kodi repo for best practices and produces a report containing found problems and warnings.
+
+It can also be use locally for detecting problems in your addons.
```
Comment (Razzeee) [inline README.md]: Used

**Pull request merged**